app/models/measure.rb

```diff
@@ -389,6 +389,12 @@ def meursing?
     measure_components.any?(&:meursing?)
   end
 
+  def zero_mfn?
+    third_country? &&
+      measure_components.count == 1 &&
+      measure_components.any?(&:zero_duty?)
```

Comment:
Hmm because the measure_components has to be an array of one element Line#394, can we not just say:

```ruby
measure_components.first.zero_duty?
```

Reply:
We could